egs2/qasr_tts/tts1/local/data_download.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Massa Baali
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+download_dir=$1
+
+# check arguments
+if [ $# != 1 ]; then
+    echo "Usage: $0 <download_dir>"
+    exit 1
+fi
+
+set -euo pipefail
+echo $download_dir
+cwd=$(pwd)
+
+if [ ! -d "${download_dir}/qasr_tts-1.0" ] && [ -f "qasr_tts-1.0.zip" ]; then
+    mkdir -p "${download_dir}"
+    cd "${download_dir}"
+    unzip "${cwd}/qasr_tts-1.0"
```

Comment:
You are unzip a folder? You can direct unzip it to another directory, no `cd` is need

Reply:
Without the `cd` I am getting the extracted file in a different directory.

Reply:
I added the .zip

Reply:
You can use `-d` to direct the output to the target directory